### ISSUE TITLE
Scroll works fine until mouseUp

### ISSUE THREAD

**Issue by chiku524** [oberonamsterdam/react-scrollbooster#11]
When I click the mouse down and drag through my content, it does in fact scroll through. However, when I let go of the mouse button (mouseup), it scrolls back to its original location (from where I started the mousedown event). 

I've tried both methods: wrapping my JSX with the component <ScrollBoost> and also using the useScrollBoost hook while setting the ref of my viewport. Both methods produce the same result.

Do I need to add another option when using the component/hook? Am I doing something wrong or is this a known bug?

Thanks

**Comment by chrisneven** [comment]
Hi @chiku524,

Can you provide a reproduction case? 

Thanks!

**Comment by chiku524** [comment]
Yes! Sorry. 

[I created a code sandbox that reproduces the issue, you can find it here](https://codesandbox.io/s/cool-sara-e2f2h?file=/src/styles.css)

_**App.js**_ is the component that displays the data, and **_styles.css_** is where I styled it.

Hope this helps and let me know if you need anything else! Still relatively new to creating github issues.

**Comment by chrisneven** [comment]
Hi @chiku524,

No problem, thanks for providing the sandbox!

So what's actually going on here is that [scrollbooster](https://github.com/ilyashubin/scrollbooster) needs a viewport element and a content element in order for it to work. I've modified your sandbox, applied the above and you'll see that it works now: https://codesandbox.io/s/blissful-cohen-4h1yb. So, hopefully that solved your problem 🙂

However, I see now that the readme isn't really clear about this and the example given in the readme actually won't work. It's on my todo list to improve the readme as it's kind of missing some important parts.

**Comment by chiku524** [comment]
Awesome! That did the trick. 

And yeah, I was using the readme for reference which is how I ended up with that predicament. I did also try reading through scrollbooster's readme but I'm guessing I just had my _content_ option pointing to a node that wasn't working well with it. No problem though :)

Anyways, thanks for helping me solve this issue in a quick manner and great package! It's very easy to use and very useful when displaying data. I will close this issue and I hope you have a great week!
